nixos/modules/services/misc/recyclarr.nix

```diff
@@ -0,0 +1,141 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.recyclarr;
+in
+{
+  options.services.recyclarr = {
+    enable = mkEnableOption "recyclarr service";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.recyclarr;
+      defaultText = literalExpression "pkgs.recyclarr";
+      description = "The recyclarr package to use.";
+    };
+
+    command = mkOption {
+      type = types.str;
+      default = "sync";
+      description = "The recyclarr command to run (e.g., sync).";
+    };
+
+    configFile = mkOption {
+      type = types.either types.path types.str;
+      description = lib.mdDoc ''
+        Path to a compatible recyclarr configuration file.
+        The file should be a valid YAML configuration following the recyclarr specification.
+        For detailed configuration options and examples, see the
+        [official configuration reference](https://recyclarr.dev/wiki/yaml/config-reference/).
+      '';
+    };
+
+    workingDir = mkOption {
```

Comment:
Is there a use case for changing it?

Reply:
If not specified, the recyclarr executable will use the directory where the binary file is located as the working directory. I assumed that if a specify location is defined, users should have the option to change it.